docs/4.3/kubernetes_ssh.md

```diff
@@ -94,18 +94,24 @@ proxy_service:
 
 ![teleport-ssh-kubernetes-integration](img/teleport-kubernetes-outside.svg)
 
-To retrieve the Kubernetes credentials for the Teleport proxy service, you have to
-authenticate against your Kubernetes cluster directly then copy the file to `/path/to/.kube/config`
- on the Teleport proxy server.
+To generate the `kubeconfig_file` for the Teleport proxy service, configure
+your `kubectl` to point at the Kubernetes cluster and use [this
+script](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh).
```

Comment:
I know this follows the pattern of our old script, but I wonder if we need to more explicit. That 
1. The user needs  must configure your local kubectl to point to the right k8s cluster and have admin-level access.
2. Download script `wget https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh` 
3. Make script writable
4. `Usage ./get-kubeconfig.sh`

Reply:
I definitely like the idea of being very explicit.

Reply:
Added, PTAL